### DISCUSSION TITLE
ef migrations build failed but dotnet build success

### DISCUSSION THREAD

**Discussion by GoldenPotato137** [microsoft/WindowsAppSDK#4062] [Q&A]
<img width="801" alt="image" src="https://github.com/microsoft/WindowsAppSDK/assets/41829668/ab6162b0-c59b-453c-84a2-c978b40b3800">

I am really confused and didn't find useful information on keyword "winui3 entity framework". Any help would be greatly appreciated.

steps to reproduce:
* Create a blank WinUI 3 app.
* Install nuget packages: Microsoft.EntityFrameworkCore.Design & Microsoft.EntityFrameworkCore.Sqlite (both v7.0.14)
* dotnet ef migrations add Init

**Comment by ghost1372** [comment]
I think you should install `Tools` package like design and sqlite

**Reply by GoldenPotato137** [reply]
Thanks for answering! 
I install Microsoft.EntityFrameworkCore.Tools (v7.0.14) and still get same error (i.e. build failed) 
Here are the installed nuget packages.
<img width="591" alt="image" src="https://github.com/microsoft/WindowsAppSDK/assets/41829668/543587b7-478e-486a-ba66-a19bb11b640f">
